libs/model/src/models/community.ts

```diff
@@ -170,7 +126,6 @@ export default (
       namespace: { type: dataTypes.STRING, allowNull: true },
       created_at: { type: dataTypes.DATE, allowNull: true },
       updated_at: { type: dataTypes.DATE, allowNull: true },
-      redirect: { type: dataTypes.TEXT, allowNull: true },
```

Comment:
Is there a reason we are removing this - is it a faulty merge conflict resolution or intended?

Reply:
Yeah that must have been a mad merge resolution. Thanks for pointing it out